danielMunteanu/src/main/java/danielCollectionTask/UserOrder.java

```diff
@@ -0,0 +1,12 @@
+package danielCollectionTask;
+
+import java.util.List;
+
+
+public class UserOrder extends AbstractOrder {
+
+
```

Comment:
user order should have one more distinct attribute of type User, otherwise we we are creating a class which is the same as the parent one?

Reply:
fixed next amend